clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -56,12 +56,33 @@ trait IoUtil {
     gs.exists(path) == 0
   }
 
+  def getMd5HashOfGoogleObject(path: String): String = {
+    new GsUtil(None).md5Hash(path)
+  }
+
+  def getSizeOfGoogleObject(path: String): Long = {
+    new GsUtil(None).du(path).head.split(" +").head.toLong
```

Comment:
This logic should be moved inside `GsUtil`

Reply:
I think we were trying to keep `GsUtil` as general as possible. GsUtil could probably be an inner class to IoUtil. I don't think anything uses it without going through GsUtil.